crates/bevy_window/src/window.rs

```diff
@@ -5,7 +5,7 @@ use bevy_utils::{tracing::warn, Uuid};
 use raw_window_handle::RawWindowHandle;
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, Reflect, FromReflect)]
-#[reflect_value(PartialEq, Hash)]
+#[reflect_value(Debug, PartialEq, Hash)]
```

Comment:
Does this need to be `reflect_value`?

Reply:
I left it as is. I'm guessing that `#[reflect_value]` was used originally to avoid exposing the private field. Do you think it makes sense to change it to `#[reflect]`, and if so, would we also want to expose the `Uuid` that it uses internally?

Reply:
Hm, you might be right. Seeing as it's an ID type (and I'm assuming it's meant to be opaque), it might be best to leave this as is.